src/pages/TaskRun/TaskRun.vue

```diff
@@ -212,6 +212,7 @@ export default {
         :style="{
           display: $vuetify.breakpoint.smAndDown ? 'inline' : 'block',
           width: taskRunNameLength + 'ch',
+          minWidth: '10vw',
```

Comment:
Can we make a css class for this? That way we're not binding an inline style for something that will never change.